crates/core/component/funding/src/component/state_key.rs

```diff
@@ -1,3 +1,16 @@
-pub fn funding_parameters() -> &'static str {
+pub fn staking_funding_parameters() -> &'static str {
```

Comment:
I think we should have this be unchanged, especially so that it matches the `put_funding_parameters` method